test/cli/class_of-any/class_of-any.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+main/sorbet --silence-dev-message test/cli/class_of-any/class_of-any.rb 2>&1
```

Comment:
Would you mind cargo-culting this test:

https://github.com/sorbet/sorbet/blob/master/test/cli/suggest_t_must/suggest_t_must.sh

So that we can test the autocorrect codepaths as well?